### ISSUE TITLE
Text selection not visible on identifiers

### ISSUE THREAD

**Issue by ishax-kos** [whizkydee/vscode-palenight-theme#88]
![Screenshot 2023-08-03 143310](https://github.com/whizkydee/vscode-palenight-theme/assets/25571687/2fb73830-36cc-4508-813f-81a17b52a5c4)
In this screenshot the letters "int" are selected inside of "printf" but after a small delay, the identifier is highlighted in full in a darker color, and the actual selected text is not distinguishable.
![Screenshot 2023-08-03 143655](https://github.com/whizkydee/vscode-palenight-theme/assets/25571687/693e256a-6a57-49ee-b7ee-211369ef4efb)
In this second image, the "t" in "return" is selected, and everything is visible.

**Comment by ishax-kos** [comment]
![image](https://github.com/whizkydee/vscode-palenight-theme/assets/25571687/995691e7-3551-4aef-8792-4e22171b3737)
here is the first screenshot taken with the monokai theme. The cursor and selection is identical.

**Comment by projeffboy** [comment]
I have a similar issue but with text in the terminal. Selecting text there, I can barely tell that what part of the text I am selecting:
<img width="232" alt="Screenshot 2024-09-09 at 8 53 53 PM" src="https://github.com/user-attachments/assets/47533dd0-c3ff-48c8-9230-e905f3bf5a2e">

**Comment by whizkydee** [comment]
Hey @ishax-kos, I'm unable to reproduce this in `v2.0.3`. Is this still an issue?

**Comment by whizkydee** [comment]
@projeffboy I agree it's almost impossible to identify what's been highlighted in the terminal. I've created #92 to track this issue separately.

**Comment by ishax-kos** [comment]
@whizkydee It seems it still occurs. Do you have the setting 'Editor: Occurrences Highlight' turned off?
